cfe_internal/enterprise/federation/federation.cf

```diff
@@ -208,18 +247,14 @@ bundle agent transport_user
       handle => "ssh_config_configured",
       edit_line => default:insert_lines("IdentityFile $(ssh_priv_key)");
 
-    selinux_enabled.incorrect_ssh_context::
-      "$(ssh_home_cil_file)"
-        create => "true",
-        edit_defaults => default:empty,
-        edit_line => default:insert_lines('(filecon "$(home)/.ssh" any (system_u object_r ssh_home_t ((s0)(s0))))');
+  methods:
+    "semanage_installed" usebundle => semanage_installed;
 
   commands:
     # _stdlib_path_exists_<command> and paths.<command> are defined is masterfiles/lib/paths.cf
-    selinux_enabled.incorrect_ssh_context.default:_stdlib_path_exists_semodule.default:_stdlib_path_exists_restorecon::
-      # the filename becomes the name for the selinux policy (as listed by semodule -l)
-      "$(paths.semodule) -i $(ssh_home_cil_file)";
-      "$(paths.restorecon) -R -F  $(home)/.ssh/";
+    selinux_enabled.incorrect_ssh_context.default:_stdlib_path_exists_semanage.default:_stdlib_path_exists_restorecon.selinux_makefile_present::
+      "$(paths.semanage) fcontext -a -t ssh_home_t $(home)/.ssh";
```

Comment:
The path here is a regex and should cover subdirectories and files inside the directory as well. So it should end with `(/.*)?`.

Reply:
right. That's what I had before but hadn't made it all the way to testing this specifically yet. Thanks. :+1:

Reply:
@vpodzime fixed